qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -364,8 +370,9 @@ def _set_qubit_x_true(clifford, circuit, qubit):
     This is done by permuting columns l > qubit or if necessary applying
     a Hadamard
     """
-    x = clifford.destabilizer.X[qubit]
-    z = clifford.destabilizer.Z[qubit]
+    # get destabilizer
+    x = clifford.paulis.x[qubit]
+    z = clifford.paulis.z[qubit]
```

Comment:
This change doesn't look equivalent.  If there's a bug here, it'd be good to put in a test to ensure it doesn't happen again.

Reply:
Since destabilizer is in the first half of Paulis, this is equivalent. However, it may be difficult to understand. The performance will be worse for the slices, but I will change it to use destabilizers, which are easier to read.